smartcar/types.py

```diff
@@ -204,6 +204,8 @@ def make_access_object(access: dict) -> Access:
     [("webhook_id", str), ("vehicle_id", str), ("meta", namedtuple)],
 )
 
+Response = NamedTuple("Response", [("body", dict), ("meta", dict)])
```

Comment:
```suggestion
Response = NamedTuple("Response", [("body", dict), ("meta", namedtuple)])
```
we should be able to use the Meta namedtuple here just like the other response types